.github/workflows/ci2.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 2 * * *' # Run everyday, at 2AM UTC.
+name: ci-profiler
+jobs:
+  system-test:
```

Comment:
Looking at my PR history, I think this was done to start running all tests with Node 13 a bit earlier (https://github.com/googleapis/cloud-profiler-nodejs/issues/585).
It may have also improve cove coverage results.

Reply:
Got it - so these are kinda 2 different things:
- We wanted node 13 testing faster here than we rolled it out elsewhere.  Moving forward, we just need to be far more bullish about getting these in the matrix as soon as they're released.  Good news is that 15 rolled out super fast. 
- For system tests - I just don't want the duplicate config here to run it as an action unless we have a goodish reason :)